replicatedhq/ship
# Run kustomize with base resources

What I Did
------------
- Run kustomize with base resources

How I Did it
------------
- Run kustomize with base resources
- Move splitting of multidoc yaml to kustomize step and also deal with base resources recursively
- CRDs are left in their original state

How to verify it
------------
- Tests, Integration
- Run `ship init` and check the diff in the UI

Description for the Changelog
------------
- Run kustomize with base resources


Picture of a Boat (not required but encouraged)
------------
🚢
